Swagger now displays descriptions and info about parameters

Letaguator opened the pull request:
Added the generation of XML documentation, and enabled such that swagger reads the xml documentation and displays it accordingly.

Letaguator commented on backend/backend.csproj:
```diff
@@ -4,6 +4,9 @@
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DocumentationFile>bin\Debug\netcoreapp3.1\backend.xml</DocumentationFile>
```
For the reviewers: this compile flag ensures that xml documentation is generated!

AndreasJJ commented:
It's still missing description for GET and PUT at /api/Metadata, GET at /api/MetadataType, GET at /api/Municipality, GET and PUT at /api/Tag, POST GET PUT at /api/User/auth

Letaguator commented:
So far no descriptions have been added yet, it currently only enables displaying of descriptions.

AndreasJJ commented:
Might as well add them so that the pull request can be connected to #61 ?

Letaguator commented:
Thought about that, but it can cause a conflict with the work I did in another branch, where I updated documentation on methods I reworked. See #72. So might be best to merge this, and add documentation when reworking methods. And do a more thorough commenting round after all of these Pull Requests/branches are merged?

AndreasJJ commented:
Well, that's why you shouldn't change stuff that's outside the scope of the issue, but its whatever as long as its done before the sprint is over.

Letaguator commented:
I know, since I changed the functionality it was within the scope on #72 as far as I know. I have now started adding documentation to all of the new functionality I add.
